src/main/resources/data.sql

```diff
@@ -8,17 +8,17 @@ TRUNCATE TABLE `identity_roles`;
 INSERT INTO `client` (id, active, uid, password)
 VALUES (1, true, '9fbd4ae2-2db3-44c7-9544-88e80255b56e', '$2a$10$AbxhLGtIx7yv8jhF0BePiOxnb1mlHHq/Ge4R3PxCL2wIsoEov1VaS');
 
-INSERT INTO `role` (id, name) VALUES
-(1, 'USER'),
-(2, 'SUPER_USER');
+-- INSERT INTO `role` (id, name) VALUES
+-- (1, 'USER'),
+-- (2, 'SUPER_USER');
```

Comment:
Why remove these? If the ID is screwing you up change the queries to support using autoincrement, e.g. `INSERT INTO role (name) VALUES ('USER')`